tests/telemetry/events/swss_events.py

```diff
@@ -67,6 +78,10 @@ def shutdown_interface(duthost):
 
 def generate_pfc_storm(duthost):
     logger.info("Generating pfc storm")
+    nslist = duthost.get_asic_namespace_list()
+    ns = random.choice(nslist)
+    if ns is not None:
+        ns = "asic0"
```

Comment:
Just want to double confirm if it's intentional to reset random namespace to "asic0" here.

Reply:
No. We should not need to always stick to asic0. Removed.